scripts/setLivenessPackagePrivate.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * This is a script that runs before publish latest and sets the `@aws-amplify/ui-react-liveness`
+ * package.json private field to true so that it is not published by changesets cli publish.
+ */
+
+const fs = require('fs');
+
+// Read the package.json file
+const packageJson = require(`../packages/react-liveness/package.json`);
+
+// Set the "private" field to true
+packageJson.private = true;
+
+// Write the updated package.json file
+fs.writeFileSync(
+  './packages/react-liveness/package.json',
```

Comment:
`writeFileSync` and `require` have different paths because require is relative and writeFileSync seems to be run from the directory the script is fun from. Not sure if this will work the same in gh action but I tested locally